src/style.css

```diff
@@ -0,0 +1,133 @@
+/* Font Roboto 400 500 700*/
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+/* CSS */
+* {
+  box-sizing: border-box;
+}
+
+h2,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+}
+
+.product {
+  padding: 34px 16px 16px;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.product__img {
+  display: block;
```

Comment:
isn't it a default style for img tag?

Reply:
if I don't add this style, my tests don't pass because there is an empty space under the picture